src/common/__tests__/Common.test.ts

```diff
@@ -129,4 +140,162 @@ describe('Commons utils tests', () => {
     test('Test that iOS platform check does not match empty input string', async () => {
         expect(common.CommandLineUtils.platformFlagIsIOS('') === false);
     });
+
+    test('Platform flag config property returns expected flag', async () => {
+        let platformFlagConfig = common.CommandLineUtils.createFlagConfig(
+            common.FlagsConfigType.Platform,
+            true
+        );
+        expect(platformFlagConfig.platform).toBeDefined();
+        expect(platformFlagConfig.platform!.longDescription).toBe(
+            messages.getMessage('platformFlagDescription')
+        );
+        expect(platformFlagConfig.platform!.description).toBe(
+            messages.getMessage('platformFlagDescription')
+        );
+        let requiredKeyValuePair = Object.entries(
+            platformFlagConfig.platform!
+        ).find((keyValuePair) => keyValuePair[0] === 'required');
```

Comment:
I need to search through the object because `required` property is not exposed through TypeScript.